tests/virt_autotest/virt_utils.pm

```diff
@@ -302,4 +302,97 @@ sub upload_supportconfig_log {
     save_screenshot;
 }
 
+# Download guest image and xml from a NFS location to local
+# the image and xml is coming from a guest installation testsuite
+# need set SKIP_GUEST_INSTALL=1 in the test suite settings
+# only available on x86_64
+sub download_guest_assets {
+
+    # guest_pattern is a string, like sles-11-sp4-64, may or may not with pv or fv given.
+    my ($guest_pattern) = @_;
+    #to handle the case: suite setting of sles-15 may be sles-15-sp0, while its image is named sles-15-fcs
+    $guest_pattern =~ s/sles-15-sp0/sles-15-fcs/i;
+
+    # list the guests matched the pattern
+    my $qa_guest_config_file = "/usr/share/qa/virtautolib/data/vm_guest_config_in_vh_update";
+    my $cmd                  = "grep \"VM_NAME=.*$guest_pattern.*\" $qa_guest_config_file | cut -d '=' -f2";
+    if (get_var('SYSTEM_ROLE', '') eq "kvm") {
+        $cmd .= " | grep -v \'\\-pv-\'";
+    }
+    my $guest_list = script_output $cmd;
+    save_screenshot;
+
+    # mount the remote NFS location of guest assets
+    # OPENQA_URL="localhost" in local openQA instead of the IP, so the line below need to be turned on and set to the webUI IP when you are using local openQA
+    # Tips: Using local openQA, you need "rcnfsserver start & vi /etc/exports; exportfs -r")
+    set_var('OPENQA_URL', "10.67.18.247");
```

Comment:
I guess this is here for testing purposes... Right? :)

Reply:
yes, forgot to comment it out :(